lib/sprite_animation.dart

```diff
@@ -4,6 +4,81 @@ import 'dart:ui';
 import 'extensions/vector2.dart';
 import 'sprite.dart';
 
+class SpriteAnimationFrameData {
+  /// Coordinates of the sprite of this Frame
+  final Vector2 srcPosition;
+
+  /// Size of the sprite of this Frame
+  final Vector2 srcSize;
+
+  /// The duration to display it, in seconds.
+  double stepTime;
```

Comment:
why are the others final and not this?

Reply:
not sure, I probably just copied this from the previous implementation, but I changed it to final.